guides/plugins/apps/content/cms/add-custom-cms-blocks.md

```diff
@@ -20,36 +20,38 @@ The basic directory structure looks as follows:
 
 ```text
 ├── Resources
+│   ├── app
+│   │   └── storefront
+│   │       └── src
+│   │           └── scss
+│   │               └── base.scss
 │   ├── cms
 │   │   └── blocks
-│   │       ├── my-first-block
+│   │       ├── swag-image-text-reversed
 │   │       │   ├── preview.html
 │   │       │   └── styles.css
-│   │       └── my-second-block
-│   │           ├── preview.html
-│   │           └── styles.css
 │   ├── views
 │   │   └── storefront
-│   │       └── cms
-│   │           └── block
-│   │               ├── my-first-block.html.twig
-│   │               └── my-second-block.html.twig
+│   │       └── block
+│   │           └── cms-block-swag-image-text-reversed-component.html.twig
 │   └── cms.xml
 └── manifest.xml
 ```
 
 Each CMS block defined within your `cms.xml` must have a directory matching the block's name in `Resources/cms/blocks/`.
-In those directories you shape your blocks by supplying a `preview.html` containing the template used for displaying a preview.
-Styling the preview is possible from the `styles.css`.
+In those directories you shape your blocks for the CMS editor in the Administration by supplying a `preview.html` containing the template used for displaying a preview.
+Styling the preview in the sidebar and the component on the CMS stage is possible from the `styles.css`.
 
 {% hint type=info %}
 Due to technical limitations it's not possible to use templating engines \(like Twig\) or preprocessors \(like Sass\) for rendering and styling the preview.
 {% endhint %}
 
+The Storefront representations reside in the `Resources/views/storefront/block/` directory.
+
 ## Defining blocks
 
 As already mentioned above and similarly to an app's `manifest.xml` CMS blocks also require some definition which is done in a `cms.xml`.
```

Comment:
```suggestion
As already mentioned above and similarly to an app's `manifest.xml`, CMS blocks also require some definition which is done in a `cms.xml`.
```
This part seems a bit out-of-order for me as well.

Reply:
Where would you expect it? Or is the information redundant to you?

Reply:
It's not redundant but you start by talking about the `cms.xml` and then continue with this part about the `manifest.xml`...I was asking myself where the `manifest.xml` even was mentioned to beginn with. 

So maybe we could write a short part about the `manifest.xml` or link to the other site where it is mentioned.

Reply:
I don't want to cover how to set up a `manifest.xml` in this article, I just state that "just as you already know, like for the app itself, provide the configuration as XML".

If one has never set up a `manifest.xml` previously, they won't get to the point where they could actually add a CMS block 😄

Reply:
Alright :) But pelase add the comma 😄